store/sqlstore/post_store.go

```diff
@@ -468,52 +468,48 @@ func (s *SqlPostStore) GetPosts(channelId string, offset int, limit int, allowFr
 	// Caching only occurs on limits of 30 and 60, the common limits requested by MM clients
 	if allowFromCache && offset == 0 && (limit == 60 || limit == 30) {
 		if cacheItem, ok := s.lastPostsCache.Get(fmt.Sprintf("%s%v", channelId, limit)); ok {
-			if s.metrics != nil {
-				s.metrics.IncrementMemCacheHitCounter("Last Posts Cache")
```

Comment:
Here we were counting hits, so you should keep this one. The other two moved out is what I expected :)